chapters/publication.tex

```diff
@@ -99,91 +174,259 @@ \section{Collaborating on academic writing}
 in a format you can manage and control.\cite{flom2005latex}
-The purpose of this setup, just like with other synced folders,
-is to avoid there ever being more than one master copy of the document.
-This means that people can edit simultaneously without fear of conflicts,
-and it is never necessary to manually resolve differences in the document.
 Finally, \LaTeX\ has one more useful trick:
-if you download a journal styler from the \textbf{Citation Styles Library}\sidenote{
-\url{https://github.com/citation-style-language/styles}}
-and use \textbf{\texttt{pandoc}},\sidenote{\url{http://pandoc.org/}}
-you can translate the raw document into Word by running the following code from the command line:
+using \textbf{\texttt{pandoc}},\sidenote{
+  \url{http://pandoc.org/}}
+you can translate the raw document into Word
+(or a number of other formats)
+by running the following code from the command line:
 
 \codeexample{pandoc.sh}{./code/pandoc.sh}
 
+\noindent The last portion after \texttt{csl=} specifies the bibliography style.
+You can download a CSL (Citation Styles Library) file\sidenote{
+  \url{https://github.com/citation-style-language/styles}}
+for nearly any journal and have it applied automatically in this process.
 Therefore, even in the case where you are requested to provide
 \texttt{.docx} versions of materials to others, or tracked-changes versions,
 you can create them effortlessly,
-and use Word's compare feature to generate a single integrated tracked version.
+and use external tools like Word's compare feature
+to generate integrated tracked versions when needed.
+
+\subsection{Getting started with \LaTeX\ via Overleaf}
+
+\LaTeX\ is a challenging tool to get started using,
+but the control it offers over the writing process is invaluable.
+In order to make it as easy as possible for your team
+to use \LaTeX\ without all members having to invest in new skills,
+we suggest using the web-based Overleaf implementation as your first foray into \LaTeX\ writing.\sidenote{
+  \url{https://www.overleaf.com}}
+While the Overleaf site has a subscription feature that offers some useful extensions,
+its free-to-use version offers basic tools that are sufficient
+for a broad variety of basic applications,
+up to and including writing a complete academic paper with coauthors.
+
+Overleaf's implementation of \LaTeX\ is suggested here for several reasons.
+Since it is completely hosted online,
+it avoids the inevitable troubleshooting of setting up a \LaTeX\ installation
+on various personal computers run by the different members of your team.
+It also automatically maintains a single master copy of the document
+so that different writers do not create conflicted or out-of-sync copies,
+and allows inviting collaborators to edit in a fashion similar to Google Docs.
+Overleaf also offers a basic version history tool that avoids having to use separate software.
+Most importantly, it provides a ``rich text'' editor
```

Comment:
What is _rich text editor_ here? That you have buttons you can click for bold and italic? Or that the output is automatically updated?

Reply:
Both, basically an "almost" WYSIWYG (because, of course, it is not that)